run.sh

```diff
@@ -58,11 +59,11 @@ cp /opt/instana/agent/etc/instana/com.instana.agent.main.config.Agent.cfg.templa
 cat /root/mvn-settings.xml.tmpl | gomplate > /opt/instana/agent/etc/mvn-settings.xml
 cat /root/com.instana.agent.main.sender.Backend.cfg.tmpl | gomplate > \
   /opt/instana/agent/etc/instana/com.instana.agent.main.sender.Backend.cfg
+cat /root/com.instana.agent.bootstrap.AgentBootstrap.cfg.tmpl | gomplate > \
+  /opt/instana/agent/etc/instana/com.instana.agent.bootstrap.AgentBootstrap.cfg
 cat /root/com.instana.agent.main.config.UpdateManager.cfg.tmpl | gomplate > \
   /opt/instana/agent/etc/instana/com.instana.agent.main.config.UpdateManager.cfg
 
-echo "origin = public_docker" >> /opt/instana/agent/etc/instana/com.instana.agent.bootstrap.AgentBootstrap.cfg
```

Comment:
@mdonkers why removing this?

Reply:
Because its now directly defined in the template. And them it's also certain this line is not accidentally pasted at the end of the "version pinning" SHA. Before we didn't have the template.